src/components/Picker/PickerPropTypes.js

```diff
@@ -36,13 +36,32 @@ const propTypes = {
 
     /** An icon to display with the picker */
     icon: PropTypes.func,
+
+    /** Size of a picker component */
+    size: PropTypes.oneOf(['normal', 'small']),
 };
 const defaultProps = {
     useDisabledStyles: false,
     disabled: false,
     placeholder: {},
     value: null,
-    icon: () => <Icon src={DownArrow} />,
+    icon: size => (
+        <>
+            {size === 'small'
+                ? (
+                    <Icon
+                        width={12}
```

Comment:
@shawnborton should we set constants for these?

Reply:
Or should they be in the pickerstyles?

Reply:
+1 I think we put all styles inthe styles file, no?

Reply:
Also why we need this conditional, shouldn't that be handled [here](https://github.com/Expensify/App/pull/4134/files#diff-7e86f78bbc185e4520a5328a405ee39beb69b81e2ae6dd0c6de2d31c916f7720R19-R23)?

Reply:
> should we set constants for these?

added `styles.pickerSmall.icon.{width,heigth}` and now using those instead of hardcoded values

> Also why we need this conditional, shouldn't that be handled here?

Since this Picker component supports overwriting the Icon component via props and this is not a required prop, the default implementation for it is in `Picker/PickerPropTypes.js`. If we were do this logic in `Picker/index.js` we kinda have to ditch the `icon` prop entirely since it will be overwritten inside the component anyway. I too feel like it doesn't look good, but don't know if we can do it cleaner. Maybe we should indeed drop support for icon overwrite since it doesn't seem to be used, but I think we should look into why it was implemented like that in the first place, maybe it is required for some future feature?